### ISSUE TITLE
White frame around images

### ISSUE THREAD

**Issue by korontari** [backdrop-contrib/masonry_views#4]
I noticed that there is always a white frame around images and I didn't find the way to control/remove it. It seems like it doesn't depend on current theme and is built into the module. I think this should be controlled by a theme or at least there should be an option to turn the frame off or set it to zero width.

![2018-12-04_11-20-37](https://user-images.githubusercontent.com/44873997/49428624-9ecb4e80-f7b7-11e8-8af0-347f8f00ddb7.png)

**Comment by laryn** [comment]
@korontari Yes, this relates to the default CSS that is used. When we fix the other issue I think we can probably wrap this into it.
